docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using the *Create project* wizard.
```

Comment:
Update this to:

You can create a project in the {WebName} with the Create project wizard.